### ISSUE TITLE
Add blt so we can use ads

### ISSUE THREAD

**Issue by rlnorthcutt** [acquia/df#161]
Acquia Developer Studio requires BLT for managing codebases, so we need to include it.

**Comment by rlnorthcutt** [comment]
I tried just adding plain BLT 9composer require), but it creates problems with composer. Mike M. suggests that it is because BLT has its own requirement for Lightning in addition to our own:
'delete (manually) the line in your composer.json file or lines for acquia/lightning and any of its subs'

**Comment by brantwynn** [comment]
Yeah dude it's a whole can of worms check out https://github.com/acquia/demo-launcher
You don't actually declare BLT in your distro you'd create a project that included ads with BLT declaring DF as the dependency.

**Comment by brantwynn** [comment]
I think we can close this as not a fit for DF.  You need a project that includes BLT + DF instead of the default BLT that comes with Lightning.
